src/main/java/race/InputView.java

```diff
@@ -0,0 +1,42 @@
+package race;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+    private final Scanner SCANNER = new Scanner(System.in);
+    private final String CAR_NAME_SPLIT_REGEX = "\\s*,\\s*";
+
+    public List<String> receiveCarNames() {
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        String carNamesString = SCANNER.nextLine();
+        List<String> carNames = List.of(carNamesString.split(CAR_NAME_SPLIT_REGEX));
+        return carNames;
```

Comment:
```suggestion
        return List.of(carNamesString.split(CAR_NAME_SPLIT_REGEX));
```
inline으로 처리할 수 있어보여요

Reply:
```java
return Arrays.stream(carNamesString.split(","))
                .map(String::trim)
                .collect(Collectors.toUnmodifiableList());
```

정규식은 아무래도 한번 더 들여다봐야 알 수 있다고 생각합니다.
그래서 문자열을 간단히 처리할 수 있다면 위 방법도 좋다고 생각해요
더 간단해 보이지 않나요?

Reply:
말씀하신 게 훨씬 깔끔한 것 같습니다. 
가독성이 좋으면서 깔끔한 코드를 만들기 위해 고민 많이 해보도록 하겠습니다!